frontend/packages/console-shared/src/components/lists/DetailPropertyListItem.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { ListItem } from '@patternfly/react-core';
+
+type Props = {
+  title?: string;
+};
+
+const DetailPropertyListItem: React.FC<Props> = ({ title, children }) => (
+  <ListItem>
+    {title && <span className="co-detail-property-list__item-title">{title}: </span>}
```

Comment:
missing `import './DetailPropertyList.scss';` import. There would be an issue with missing styles if we use `DetailPropertyListItem` without `DetailPropertyList`

Reply:
Done